detekt.yml

```diff
@@ -24,25 +24,6 @@ console-reports:
   #  - 'FindingsReport'
   #  - 'BuildFailureReport'
 
-comments:
```

Comment:
@wangerekaharun I can see the file was auto edited actually :)

So I guess were good to go

Reply:
Interesting, never knew that it does so.